analyzers/stylecop.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
+  "settings": {
+    "orderingRules": {
+      "systemUsingDirectivesFirst": true,
+      "usingDirectivesPlacement": "outsideNamespace",
+      "elementOrder": [
+        "constant",
+        "static",
```

Comment:
We should ignore the `static` configuration, as it's noisy for private static methods in our code base.
```suggestion
```

Reply:
yes, and I will also remove accessibility, because we can have `private static` at the top